.rubocop.yml

```diff
@@ -18,17 +18,10 @@ AllCops:
   - "**/Puppetfile"
   - "**/Vagrantfile"
   - "**/Guardfile"
-Metrics/LineLength:
+Layout/LineLength:
+  Enabled: false
```

Comment:
Disabled this cop because of https://github.com/puppetlabs/puppetlabs-host_core/blob/main/lib/puppet/type/host.rb#L21 ...

Reply:
![20210924_171328.jpg](https://user-images.githubusercontent.com/25789827/134768346-7d6ad1f3-e617-47f0-8e7d-fb1057a96299.jpg)

Reply:
I think we can also disable the cop on that specific line by wrapping it using the comment syntax: https://stackoverflow.com/a/54987958/6648276